tools/kernel-configs/microvm-kernel-aarch64-4.14.config

```diff
@@ -1413,6 +1413,7 @@ CONFIG_PTP_1588_CLOCK=y
 #
 # Enable PHYLIB and NETWORK_PHY_TIMESTAMPING to see the additional clocks.
 #
+CONFIG_PTP_1588_CLOCK_KVM=y
```

Comment:
Maybe `CONFIG_PTP_1588_CLOCK=y` is needed as well?
https://github.com/firecracker-microvm/firecracker/blob/d79b7d456db28e8a34b174a48067da2376a63f14/FAQ.md?plain=1#L147-L163

Reply:
That is already present in the file line [1411](https://github.com/firecracker-microvm/firecracker-containerd/pull/736/files#diff-cc930498ee702eba4cc58ef512ba11b1b3a7c31e27b54ec850585aefdb1fb597R1411)